src/test/java/org/apache/drill_web_test_framework/web_ui/tests/options/OptionsTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill_web_test_framework.web_ui.tests.options;
+
+import org.apache.drill_web_test_framework.web_ui.steps.BaseSteps;
+import org.apache.drill_web_test_framework.web_ui.steps.NavigationSteps;
+import org.apache.drill_web_test_framework.web_ui.steps.OptionsSteps;
+import org.apache.drill_web_test_framework.web_ui.tests.FunctionalTest;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static org.testng.Assert.assertTrue;
+
+public class OptionsTest extends FunctionalTest {
+  private final OptionsSteps optionsSteps = BaseSteps.getSteps(OptionsSteps.class);
+  private final NavigationSteps navigationSteps = BaseSteps.getSteps(NavigationSteps.class);
+
+  @Test
+  public void sortingByPlanner() {
+    String planner = "planner";
+    List<List<String>> table = navigationSteps.navigateOptions().selectPlannerFilter().getOptionsTable();
+    assertTrue(optionsSteps.isSearchFieldFilled(planner));
+    table.forEach(x -> {
+      if (x != table.get(0)) {
```

Comment:
You can use .filter() method from Java 8 instead of "if" statement here.

Reply:
Not filter() but skip().

Reply:
Both methods will get the same result in a different way.
But in this case, skip() will make the code more readable, thanks for your catch.

Reply:
Done